assets/queries/cloudFormation/filter_and_alarm_exist_for_route_table_changes/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_Route_Table_Changes",
+  "queryName": "Filter And Alarm Not Exists For Route Table Changes",
+  "severity": "MEDIUM",
+  "category": "Monitoring",
+  "descriptionText": "A log metric filter and alarm should exist for route table changes",
+  "descriptionUrl": "https://kb.securestate.vmware.com/monitor-route-table-changes.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline